doc(xero): source package hyperlink update

fivetran-abhijeet opened the pull request:
The source package link on the transform package readme is broken. 

![image](https://user-images.githubusercontent.com/60129976/115339989-82cb7a80-a1c3-11eb-970e-c72592853657.png)

fivetran-kristin reviewed (commented):
Thanks! I added one suggestion, which is to remove the reference to a specific version of the package, but rather just link to the repo's main page.

fivetran-abhijeet commented:
> Thanks! I added one suggestion, which is to remove the reference to a specific version of the package, but rather just link to the repo's main page.

Thanks @fivetran-kristin. Incorporated the changes.

fivetran-kristin reviewed (approved):
LGTM
